lib/easybill-rest-client/generic_api.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+module EasybillRestClient
+  class GenericApi
```

Comment:
Yes, it would.
I was trying to make it super-dry and short. Also, there's a benefit that you don't have to know about existence of `resource_name` and `resource_class`. But I'm fine with either way.